test/specs/platform.tests.js

```diff
@@ -0,0 +1,17 @@
+describe('Chart.platform', function() {
+	it('the default platform is dom', function() {
+		expect(Chart.platform.current.type).toEqual('dom');
```

Comment:
it seems like `type` is only used in the tests. I think if you switch to ES6 classes you can just check if it's an instance of the expected type instead of having to add a new `type` field